containerfixture/containerfixture.go

```diff
@@ -131,10 +150,41 @@ func cleanup(t *testing.T, podName string) {
 }
 
 func WithServiceContainer(t *testing.T, ServiceManifest string, testFunc func(FixtureInfo)) {
+	rootDir, err := scripts.GetRepoRootDir()
+	if err != nil {
+		t.Errorf("Could not determine path for building the container")
+		t.Fail()
+	}
+
+	images := getManifestImages(t, ServiceManifest)
+	for _, image := range images {
+		buildContainer(t, image, rootDir)
+	}
 	deployment := buildAndRunPod(t, ServiceManifest)
 
 	defer cleanup(t, deployment.PodName)
-
-	testFunc(deployment)
 	return
 }
+
+type PodDefinition struct {
```

Comment:
@Omeramsc I'm not convinced these annotations are needed since AFAIK the YAML and JSON readers know how to assume the field names are lower case in the data. Did you try running the code without them?

Reply:
It does seem to work without the annotation, I understood differently from the [documentation](https://pkg.go.dev/gopkg.in/yaml.v2#Unmarshal) though. 
anyway, fixed that.